hawkbit-repository/hawkbit-repository-jpa/src/main/resources/db/migration/SQL_SERVER/V1_12_16__add_action_initiated_by___SQL_SERVER.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE sp_action ADD initiated_by VARCHAR(64) NOT NULL;
+ALTER TABLE sp_target_filter_query ADD COLUMN auto_assign_initiated_by VARCHAR(64);
```

Comment:
I guess it should be ADD instead of ADD COLUMN for SQL Server, could you check it?

Reply:
good catch!